src/common/config_opts.h

```diff
@@ -1433,6 +1433,8 @@ OPTION(rgw_swift_versioning_enabled, OPT_BOOL, false) // whether swift object ve
 
 OPTION(rgw_list_bucket_min_readahead, OPT_INT, 1000) // minimum number of entries to read from rados for bucket listing
 
+OPTION(rgw_getusage_dump_description, OPT_BOOL, true) // dump description of total stats for s3 GetUsage API
```

Comment:
@weiqiaomiao let's rename it to rgw_rest_getusage_op_compat, and default it to false (switch the flag's meaning)